js/index.js

```diff
@@ -0,0 +1,37 @@
+
+const replaceSVGTemplatePlaceholders = (template, data) => {
+    return template.replace(/{{(.*?)}}/g, (match, key) => {
+      key = key.replace(/^\//, '').replace(/\/$/, '');
+      const keys = key.split('/');
+      let value = data;
+      keys.forEach(k => {
+        if (value) {
+          value = value[k];
+          if (isValidDateTime(value)) {
+            value = formatDate(value);
+          }
+        }
+      });
+      return value !== undefined ? String(value) : '';
+    });
+  };
+  
+  const formatDate = (dateString) => {
```

Comment:
can we move this methods to Utility class

Reply:
Done